src/release-it-config/cli/commands/README.md

```diff
@@ -0,0 +1,6 @@
+# Commands
+
+Commands should be named as follows: `name.command.ts`.
```

Comment:
Can you provide the reference to our standards for clarity? I didn't see anything other than classes and tests referenced in https://github.com/kmuncie/silvermine-info/blob/f907bedb8951ab6417c5a0ba325af05db2867a7e/coding-standards.md#naming-3

If it's not covered in the standards, I'd like to add them, so would you mind summarizing?

Reply:
Hmm yeah, that's the only section that I'm aware of. It probably needs to be fleshed out.

For JavaScript and TypeScript, we use lowercase with words separated by dashes (kebab-case) for files that export a function, an object (or pretty much anything other than a class), or that do not export anything: 

- https://github.com/silvermine/serverless-utils/blob/master/src/get-code-version.js
- https://github.com/silvermine/event-emitter/blob/master/src/lib/find.js

If the file exports a function, the name of the file is usually the name of the function converted from camelCase to kebab-case.

For files that export classes, we use UpperCamelCase:

- https://github.com/silvermine/weighted-random-selection/blob/master/src/WeightedRandomSelection.js

Those are the only two standards that I can think of.

Since a file like `commands/help.command.ts` is exporting an anonymous function, I'd expect it to either be named `commands/help-command.ts` or `commands/help.ts`. `help-command.ts` seems a little redundant since the file is already in the `commands` folder, but some people like that context in the file name 🤷‍♂️I'm good with either.

Reply:
Were you going to update this documentation to `name-command.ts`?

Reply:
Were you going to update this documentation to `name-command.ts`?